lib/moab/storage_object.rb

```diff
@@ -231,6 +227,18 @@ def restore_object(recovery_path)
       self
     end
 
+    # @return [Integer] the size occupied on disk by the storage object, in bytes.  this is the entire moab (all versions).
+    def size
+      size = 0
+      Find.find(object_pathname) do |path|
+        if FileTest.directory?(path)
+          Find.prune if File.basename(path)[0] == '.'
+        else
+          size += FileTest.size(path)
+        end
```

Comment:
it seems that neither `.` nor `..` are included in the results (which makes sense, because then failing to prune them proactively would result in infinite loop behavior by default)... as such, what about:
```ruby
size += FileTest.size(path) unless FileTest.directory?(path) # must aggregate size across all individual files
```

instead of the explicit `if/else` above?

e.g., if i do the following on a directory called `spec/fixtures/empty`, which only contains one file, called `.keep`, i get:
```ruby
[1] pry(main)> Find.find('spec/fixtures/empty').map { |dir| dir }
=> ["spec/fixtures/empty", "spec/fixtures/empty/.keep"]
[2] pry(main)> 
```

which implies to me that pruning the implicit `.` and `..` is unnecessary.

also, i don't feel strongly at all about the comment i added above, i just thought it might be nice to explain that we can't call `size` directly on a directory.  though that's sort of implied anyway if you assume we're somewhat sensible and not writing extra code when we could've just called a `size` method on the top level dir.